File: MobileViews/App_Start/MobileViewEnginesBootstrapper.vb
Imports System.Web.Mvc
Imports MobileViews.Microsoft.Web.Mvc
 
<Assembly: WebActivator.PreApplicationStartMethod(GetType(MobileViews.App_Start.MobileViewEngines), "Start")>
Namespace MobileViews.App_Start
    Public NotInheritable Class MobileViewEngines
        Public Shared Sub Start()
            ViewEngines.Engines.Insert(0, New MobileCapableRazorViewEngine())
            ViewEngines.Engines.Insert(0, New MobileCapableWebFormViewEngine())
        End Sub
    End Class
End Namespace

File: MobileViews/App_Start/MobileViewEnginesBootstrapper.cs
using System.Web.Mvc;
using Microsoft.Web.Mvc;
 
[assembly: WebActivator.PreApplicationStartMethod(typeof(MobileViews.App_Start.MobileViewEngines), "Start")]
namespace MobileViews.App_Start {
    public static class MobileViewEngines{
        public static void Start() 
        {
            ViewEngines.Engines.Insert(0, new MobileCapableRazorViewEngine());
            ViewEngines.Engines.Insert(0, new MobileCapableWebFormViewEngine());
        }
    }
}